app/tests/integration/nimbus/test_experimenter.py

```diff
@@ -86,3 +88,28 @@ def test_takeaways(
 
     assert summary.takeaways_summary_text == expected_summary
     assert summary.takeaways_recommendation_badge_text == "Change course"
+
+
+@pytest.mark.run_once
+def test_branch_screenshot(
+    selenium,
+    create_experiment,
+):
+    summary = create_experiment(selenium)
+    branches = summary.navigate_to_branches()
+    add_button = branches.add_screenshot_buttons[0]
+    add_button.click()
+
+    image_path = os.path.join(os.getcwd(), "example.jpg")
+    branches.screenshot_image_field().send_keys(image_path)
```

Comment:
Okay so that didn't end up being so bad 👍

Reply:
Yeah, figured I'd need to somehow automate the upload dialog in Firefox, but turns out this works surprisingly simply?